src/devTools.js

```diff
@@ -138,6 +143,7 @@ function liftReducer(reducer, initialState) {
         currentStateIndex++;
       }
       stagedActions = [...stagedActions, liftedAction.action];
+      timestamps = [...timestamps, Date.now()];
       break;
```

Comment:
I wanted there to be a new starting point for timestamps after RESET, etc, similar to the INIT action.